src/parser/index.ts

```diff
@@ -107,6 +108,7 @@ const configs: Config[] = [
   ClassicPaladinHoly,
   ClassicPriest,
   ClassicShaman,
+  ClassicHunter,
```

Comment:
Update to `ClassicHunterSurvival` after other changes

Reply:
Done. I used holy paladin as model.